_includes/v23.1/known-limitations/sql-cursors.md

```diff
@@ -0,0 +1,25 @@
+CockroachDB implements SQL {% if page.name == "known-limitations.md" %} [cursor](cursors.html) {% else %} cursor {% endif %} support with the following limitations:
+
+- `DECLARE` only supports forward cursors.
+- `FETCH` supports forward, relative, and absolute variants, but only for forward cursors.
+- `BINARY CURSOR`, which returns data in the Postgres binary format, is not supported.
+- `MOVE`, which allows advancing the cursor without returning any rows, is not supported.
+- `WITH HOLD`, which allows keeping a cursor open for longer than a transaction by writing its results into a buffer, is accepted as valid syntax within a single transaction but is not supported. It acts as a no-op and does not actually perform the function of `WITH HOLD`, which is to make the cursor live outside its parent transaction.
+    - This syntax is accepted (but does not have any effect):
+        {% include_cached copy-clipboard.html %}
+        ~~~ sql
+        BEGIN;
+        DECLARE test_cur CURSOR WITH HOLD FOR SELECT * FROM foo ORDER BY bar;
+        CLOSE test_cur;
+        COMMIT;
+        ~~~
+    - This syntax is not accepted, and will result in an error:
+        {% include_cached copy-clipboard.html %}
+        ~~~ sql
+        BEGIN;
+        DECLARE test_cur CURSOR WITH HOLD FOR SELECT * FROM foo ORDER BY bar;
+        COMMIT; -- This will fail with an error because CLOSE test_cur was not called inside the transaction.
+        ~~~
+- Scrollable cursor (also known as reverse `FETCH`) is not supported.
+- [`SELECT ... FOR UPDATE`](select-for-update.html) is not supported.
+- Respect for [`SAVEPOINT`s](savepoint.html) is not supported. Cursor definitions do not disappear properly if rolled back to a `SAVEPOINT` from before they were created.
```

Comment:
There are a few other things not supported which might be worth adding here:
https://github.com/cockroachdb/cockroach/issues/41412#issuecomment-1053649403

Reply:
Thank you - I have updated the descriptions of the reverse scrolling stuff a bit, and added the GitHub links